tests/neptune/new/client/test_model_version.py

```diff
@@ -101,3 +104,21 @@ def test_wrong_parameters(self):
             init_model_version(version=None, model=None)
         with self.assertRaises(NeptuneWongInitParametersException):
             init_model_version(version="whatever", model="whatever")
+
+    @patch(
+        "neptune.new.internal.backends.neptune_backend_mock.NeptuneBackendMock.get_attributes",
+        new=lambda _, _uuid, _type: [Attribute("sys/stage", AttributeType.STRING)],
+    )
+    def test_change_stage(self):
+        exp = self.call_init()
+        exp.change_stage(stage="production")
+
+        self.assertEqual("production", exp["sys/stage"].fetch())
```

Comment:
I'd prefer either `ModelVersionStage.PRODUCTION.value` both here and in `test_protected_paths`, or `"production"` both here in `test_protected_paths` (I probably like the latter more)

Reply:
Yes, good point.